applications/dashboard/settings/class.hooks.php

```diff
@@ -725,7 +725,7 @@ public function rootController_sso_create($Sender, $Target = '') {
 
         // If an event handler didn't handle the signin then just redirect to the target.
         if (!$Handled) {
-            safeRedirect($Target, 302);
+            redirectTo($Target, 302);
```

Comment:
I'm going to update review all the redirect in my next PR that's why I did not do any further in my update.